apps/services/license-api/src/app/modules/license/license.controller.ts

```diff
@@ -0,0 +1,122 @@
+import {
+  BadRequestException,
+  Body,
+  Controller,
+  Delete,
+  Param,
+  ParseEnumPipe,
+  Post,
+  Put,
+} from '@nestjs/common'
+import { LicenseService } from './license.service'
+import { Audit } from '@island.is/nest/audit'
+import { Documentation } from '@island.is/nest/swagger'
+import {
+  UpdateLicenseResponse,
+  UpdateLicenseRequest,
+  RevokeLicenseResponse,
+  VerifyLicenseRequest,
+  VerifyLicenseResponse,
+} from './dto'
+import { ApiHeader, ApiParam, ApiTags } from '@nestjs/swagger'
+import { LicenseId } from './license.types'
+import { NationalId } from '../../decorators/nationalId'
+
+@ApiHeader({
+  name: 'X-Param-NationalId',
+  description: "The user's national id",
+})
+@Controller({ version: ['1'], path: 'users/.nationalId/licenses/' })
+@ApiTags('users-licenses')
+@Audit()
+export class UserLicensesController {
+  constructor(private readonly licenseService: LicenseService) {}
+
+  @Documentation({
+    description: `The endpoint updates a single user's license. The method of update is according to the LicenseUpdateType parameter
+    If Push: The license is updated with the data provided in the payload. If Pull: The license data is pulled and used to update
+    the digital license`,
+    response: {
+      status: 200,
+      type: UpdateLicenseResponse,
+    },
+  })
+  @ApiParam({
+    name: 'License Id',
+    description: 'The license type',
+    enum: LicenseId,
+    enumName: 'LicenseId',
+  })
```

Comment:
Param can be described in the [`@Documentation`](https://docs.devland.is/libs/nest/swagger) decorator.